velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -282,9 +346,66 @@ std::shared_ptr<const core::PlanNode> SubstraitVeloxPlanConverter::toVeloxPlan(
   }
   auto outputType = ROW(std::move(outNames), std::move(veloxTypeList));
 
-  auto tableScanNode = std::make_shared<core::TableScanNode>(
-      nextPlanNodeId(), outputType, tableHandle, assignments);
-  return tableScanNode;
+  if (sRead.has_virtual_table()) {
```

Comment:
This method is very long. Would you extract a helper method to convert virtual table?

Reply:
sure.

> @ZJie1 Overall looks good, but there is no test coverage. Would it make sense to extend the velox/substrait/tests/PlanConversionTest.cpp to add tests for virtual tables?

there will need another json file to test this so extend the velox/substrait/tests/PlanConversionTest.cpp may not worked, the json file included in the code now use the scanNode. How about cover this in the round-trip test that we plan add later.